### PR TITLE
Force LF

### PR DESCRIPTION
**Pull request by anna-is-cute**
We need to choose a line-ending format. Personally, and as most people will agree, I believe LF is the best option here. This .gitattributes should convert bad line-endings to LF on push. Along with this PR, I would like to convert all CRLF files to LF, but I haven't included that, as it may break other PRs.

### CODE REVIEW

**Comment by turt2live** [comment]
Please review the CONTRI- oh.

**Comment by JeromSar** [comment]
@turt2live Go go go: https://github.com/SpaceManiac/Glowstone/pull/195

**Comment by anna-is-cute** [comment]
This should enforce #195! :D

**Comment by ewized** [comment]
:+1:

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1 @@
+text eol=lf
```
Comment (turt2live) [inline .gitattributes:1]: This comment is just to annoy @jkcclemens

Comment (anna-is-cute) [inline .gitattributes:1]: Thanks.

Comment (EntityReborn) [inline .gitattributes:1]: Secondary useless comment of doom.

**Comment by SpaceManiac** [comment]
Merged: 74efe319c929a5bcf7af8392c115e8380be6caac.

**Pull request closed without merging**